build/instrumentation.build.js

```diff
@@ -114,8 +99,12 @@ export function createConfig({
       json(),
       styles(
         mergeConfig('styles', {
-          mode: extractCss ? ['extract', outputCss] : 'inject',
-          plugins: postCSSPlugins
+          mode: [
+            'inject',
+            (varname, id) =>
+              // eslint-disable-next-line max-len
+              `import {createInjector} from 'vue-runtime-helpers';const injector=createInjector({});injector('${id}',{source:${varname}})`
+          ]
```

Comment:
I'm just curious, why do we need this now?

Reply:
to use the same way of injecting the CSS as the Vue components

Reply:
necessary to avoid issues, also for future Tailwindcss use